scripts/seed-test-data-application-team-allocation.py

```
@@ -0,0 +1,500 @@
+    # but phase_id is self-sufficient here)
+    psql(TA_CONTAINER, f"""
+        DELETE FROM allocations WHERE course_phase_id IN ({ta_phase_ids});
+        DELETE FROM student_team_preference_response
+            WHERE team_id IN (SELECT id FROM team WHERE course_phase_id IN ({ta_phase_ids}));
+        DELETE FROM student_skill_response
+            WHERE skill_id IN (SELECT id FROM skill WHERE course_phase_id IN ({ta_phase_ids}));
+        DELETE FROM tutor WHERE course_phase_id IN ({ta_phase_ids});
+        DELETE FROM team WHERE course_phase_id IN ({ta_phase_ids});
+        DELETE FROM skill WHERE course_phase_id IN ({ta_phase_ids});
+        DELETE FROM survey_timeframe WHERE course_phase_id IN ({ta_phase_ids});
+    """)
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)